src/main/java/edu/greenblitz/robotName/subsystems/arm/EndEffector/RollerUtils/RollerConstants.java

```diff
@@ -4,6 +4,7 @@ public class RollerConstants {
 
 	public static final double ROLL_FORWARD_POWER = 1;
 	public static final double ROLL_BACKWARD_POWER = -1;
+	public static final double TIME_UNTIL_NOTE_EXIT_ARM = 0.5;
```

Comment:
```suggestion
	public static final double TIME_UNTIL_NOTE_EXIT_ROLLER = 0.5;
```
sorry to do this againnnn